Sming/Services/CommandProcessing/CommandHandler.cpp

```diff
@@ -15,6 +16,8 @@
 CommandHandler::CommandHandler()
 {
 	 registeredCommands = new HashMap<String, CommandDelegate>;
```

Comment:
The indentation needs fixing.

Reply:
Do coding standards apply to 'Services'? I left the indentation as I found it...

Reply:
Line 18 and the following lines should have the same indent. In that specific case it seems that line 18 already had a wrong indentation. Just for consistency can you change line 18 to have the same indentation as line 19?